### ISSUE TITLE
Notebook_3_image_classification tweaks-critiques-changes

### ISSUE THREAD

**Issue by bryan-pardo** [interactiveaudiolab/course-deep-learning#3]
There's a lot of great info in here. I did some mild (very mild) reorganization.  The main thing I see is that it might cover too many topics in one notebook and we may want to split this up into 2 notebooks.  I'm thinking about how we want to do this....I can imagine putting everything after the first test/train loop (section 4) into a 2nd notebook that contains GPU, logging, and data augmentation. 

Also that main training loop gets repeated multiple times in different cells. I feel like there should be some way of declaring it a single time and then calling it repeatedly... 

Another issue is that the Tensorboard logger didn't display anything for me. Maybe I did something wrong. Not sure.

Also, I feel like we introduce an optimizer (SGD) but don't really talk about it and other elements (e.g loss functions) were explicitly talked about.

OK. Those were my reactions. Now we have stuff to discuss!

**Comment by bryan-pardo** [comment]
We agreed to split this notebook into 2:  The first notebook contains everything through section 4 and also GPU acceleration.   The second has augmentation and logging.....

In that first notebook, let's add a little bit of detail on our optimizer choice. A paragraph or so on SGD. Nothing big...just highlighting that it is a meaningful design choice.

**Comment by interactiveaudiolab** [comment]
Good. Thanks.
